src/Symfony/Component/Cache/Adapter/RedisTagAwareAdapter.php

```diff
@@ -234,14 +240,22 @@ protected function doInvalidate(array $tagIds): bool
         });
 
         $lua = <<<'EOLUA'
+            redis.replicate_commands()
+
             local id = KEYS[1]
             local cursor = table.remove(ARGV)
             redis.call('SREM', '{'..id..'}'..id, unpack(ARGV))
 
             return redis.call('SSCAN', '{'..id..'}'..id, cursor, 'COUNT', 5000)
 EOLUA;
 
-        foreach ($results as $id => [$cursor, $ids]) {
+        foreach ($results as $id => $values) {
+            if ($values instanceof \RedisException) {
+                throw $values;
```

Comment:
here also: better finish the foreach and return false when there is an exception